### PR TITLE
[INT-1266] Allow users in Pach enabled MLDE Notebooks to input commit id

### PR DESCRIPTION
**Pull request by emmajsadams**
* Adds a new input that allows a user to specify a commit
* Adds a display for the mountedRepoUri
* Refactors the backend to be focused on mounting commits instead of repos.

image

### CODE REVIEW

File: jupyter-extension/jupyterlab_pachyderm/pfs_manager.py [hunk 1]
```diff
@@ -200,13 +207,14 @@ def __init__(self, client: Client, **kwargs):
         self._client = client
         super().__init__(**kwargs)
 
-    def mount_branch(self, branch: pfs.Branch):
-        if not self.branch_exists(branch=branch):
-            raise ValueError("branch_uri exists but does not resolve a valid branch")
-        self.mounted_branch = branch
+    def mount_commit(self, commit: pfs.Commit):
+        print(commit)
```
Comment (bbonenfant) [inline jupyter-extension/jupyterlab_pachyderm/pfs_manager.py:211]: print

File: jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py
Comment (bbonenfant) [inline jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py:1]: I think it makes sense to add a test for mounting a specific commit. It should probably try both `project/repo@branch=commit` and `project/repo@commit`

File: jupyter-extension/style/base.css [hunk 2]
```diff
@@ -6,6 +6,10 @@
 @import url('../src/utils/components/Circle/circle.css');
 @import url('../src/utils/components/DropdownCombobox/DropdownCombobox.css');
 @import url('../src/utils/components/LoadingDots/loadingDots.css');
+@import url('../src/utils/components/TextInput/TextInput.css');
+@import url('../src/utils/components/TextInput/TextInput.css');
```
Comment (bbonenfant) [inline jupyter-extension/style/base.css:10]: added twice

**Review by bbonenfant — APPROVED** [review]
Looks Great!

File: jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py
Comment (emmajsadams) [reply jupyter-extension/jupyterlab_pachyderm/tests/test_integrations.py:1]: Will do

File: jupyter-extension/style/base.css (hunk 2 above)
Comment (emmajsadams) [reply jupyter-extension/style/base.css]: Removed

File: jupyter-extension/jupyterlab_pachyderm/pfs_manager.py (hunk 1 above)
Comment (emmajsadams) [reply jupyter-extension/jupyterlab_pachyderm/pfs_manager.py]: Removed

**Review by bbonenfant — APPROVED** [review]

**Pull request merged**